conf/core/ConfigOptions.asciidoc

```diff
@@ -1174,15 +1174,6 @@ or deleted.  It also requires writing out temporary files, so extra disk space i
 Should the hootapi services DB writer automatically create the specified user if it doesn't exist.
 This is most useful when debugging and testing.
 
-=== hootapi.db.writer.job.id
-
-* Data Type: string
-* Default Value: ``
-
-This option allows the Hootenanny Web Services to pass in the ID of the currently executing command
-job.  Given that information, any Hootenanny API database writer invoked by the command will write
-the ID of the job's associated map to the job status table.
-
```

Comment:
Changed this b/c it really doesn't have to be specific to using data out of a hoot api db.